src/test/skript/tests/syntaxes/effects/EffHealth.sk

```diff
@@ -10,14 +10,14 @@ test "health effect":
 	spawn cow at location(0, 64, 0, world "world")
 	set {_m} to last spawned cow
 	assert health of {_m} is 5 with "default cow health failed"
-	damage {_m} by 0.5
-	assert health of {_m} is 4.5 with "damage cow failed"
-	damage {_m} by 99
-	assert health of {_m} is 0 with "damage cow failed"
+	damage {_m} by 1
```

Comment:
Should be 0.5 as in the line below you also damage by an int

Reply:
oh yeah, oops